operator/src/main/resources/Operator.properties

```diff
@@ -187,6 +187,8 @@ WLSKO-0175=Job {0} in namespace {1} failed with status {2}. Check log messages \
   copied from the introspector pod {3} log for additional information.
 WLSKO-0176=Job {1} in namespace {0} failed, job details are {2}
 WLSKO-0177=Pod {0} in namespace {1} failed, the pod status is {2}
+WLSKO-0178=Patching administration server Pod labels with ''weblogic.introspectVersion'' {2} for WebLogic domain with UID: {0}. Administration server name: {1}.
+WLSKO-0179=Patching managed server Pod labels with ''weblogic.introspectVersion'' {2} for WebLogic domain with UID: {0}. Managed server name: {1}.
```

Comment:
There's a lot of overlap between the MANAGED/ADMIN _ POD_ PATCHED/CREATED/REPLACED/EXISTS log messages at KO-0120/0121/0041/0042/0043/0050/0051/0052.  

IMO, they should be aligned since that would simplify tracing pod transitions.  Something like:

```
Action on pod ''{0}'' of type ''managed server/adminstration server'', operation ''patched/created/replaced/exists'', domain UID ''{1}'', label ''weblogic.introspectVersion''=''{2}'', label ''weblogic.restartVersion''=''{3}''.
```
They could remain separate log messages, or combined into a single log message that parameterizes the operation and server type.

Also, I suspect 'domain UID' is no longer needed since it's probably always reported in the log message's domain UID field...

Reply:
I would suggest that we make this change in a different PR after everyone agrees.  Note that a lot of unit test cases check the log messages.

Reply:
I don't think all that many tests are checking _each_ message. The change shouldn't be all that much work, but I agree it is not really part of this PR.

Reply:
I've filed https://jira.oraclecorp.com/jira/browse/OWLS-85639 to track.  @rjeberhard FYI